packages/cli/src/environments/sourceControl/sourceControl.service.ee.ts

```diff
@@ -52,7 +53,7 @@ export class SourceControlService {
 		private sourceControlPreferencesService: SourceControlPreferencesService,
 		private sourceControlExportService: SourceControlExportService,
 		private sourceControlImportService: SourceControlImportService,
-		private tagRepository: TagRepository,
+		private tagService: TagService,
```

Comment:
Generally in the Controller-Service-Repository pattern, services can refer to each other, but they can also directly use repositories of other "units", especially when the service methods are simple wrappers on top of repository methods. This can help reduce cyclic dependencies between service.

![image](https://github.com/n8n-io/n8n/assets/196144/a52936d2-62c1-4969-9de7-a2a401a6df58)
 
I'll leave this for you to decide if you want to revert this bit, or not.

Reply:
Thank you! I didn't know this, reverted!

Reply:
Looks like you reverted only one of the three services in `environments/sourceControl`. Can you please revert the other two as well please 🙏🏽 .

Reply:
Done! Sorry, lost context and forgot about them!